website/docs/docs/build/packages.md

```diff
@@ -32,6 +32,21 @@ import UseCaseInfo from '/snippets/_packages_or_dependencies.md';
 
 <UseCaseInfo/>
 
+## How do I create a package?
+
+Creating packages is an advanced use of dbt, but it can be a relatively simple task. The only strict requirement is the presence of a [`dbt_project.yml file`](/reference/dbt_project.yml).
```

Comment:
```suggestion
Creating packages is an advanced use of dbt, but it can be a relatively simple task. The only strict requirement is the presence of a [`dbt_project.yml` file](/reference/dbt_project.yml).
```